components/aws/sagemaker/tests/integration_tests/utils/sagemaker_utils.py

```diff
@@ -24,3 +24,32 @@ def describe_hpo_job(client, job_name):
 
 def describe_transform_job(client, job_name):
     return client.describe_transform_job(TransformJobName=job_name)
+
+def describe_workteam(client, workteam_name):
+    return client.describe_workteam(WorkteamName=workteam_name)
+
+def list_workteams(client):
+    return client.list_workteams()
+
+def get_cognito_member_definitions(client):
+    # This is one way to get the user_pool and client_id for the Sagemaker Workforce.
+    # An alternative would be to take these values as user input via params or a config file.
+    # The current mechanism expects that there exists atleast one private workteam in the region.
+    default_workteam = list_workteams(client)['Workteams'][0]['MemberDefinitions'][0]['CognitoMemberDefinition']
```

Comment:
lets have the option to pass it via command line arguments and have this as fallback with a print message to the user

Reply:
I don't think there is any value add to it. In either case the user will have to have created the first workteam manually in the console. In this case, we are also requiring them to provide the memberdefns which we can easily read from the account.